docs/index.rst

```diff
@@ -42,6 +42,45 @@ Requirements
 
   pip3 install --user -r requirements.txt
 
+
+Installation
+------------
+
+- :py:mod:`xbout` is not currently on pip or conda. Therefore to install xbout on
+  your system you must first clone the repository using:
+
+.. code-block:: bash
+
+  git clone git@github.com:boutproject/xBOUT.git
+
+or
+
+.. code-block:: bash
+
+  git clone https://github.com/boutproject/xBOUT.git
+
+
+Once cloned navigate to the `xBOUT` directory and run the following command:
+
+.. code-block:: bash
+
+  pip3 install --user ./
```

Comment:
I think we should recommend `pip3 install --user -e .`. The `-e` (or `--editable`) means pip sets up the system so you can `import xbout`, and can provide executables (defined with `entry_points` in `setup.py`) but still uses the source files in the git repo, so if you edit them or pull a newer version it's available on the system. I've been working on getting hypnotoad2 to install with pip, and found that it could get confused if installed from the repo without the `-e` (although the problem might have been also related to mixing using and not using `-e`), and didn't properly uninstall - my best guess is that the problem was pip didn't have a version number that it could understand and compare to decide whether it should remove/update the already-installed files.

Pinging @ZedThree for a better-informed opinion...

Reply:
Or just `pip3 install git+git://github.com/boutproject/xBOUT.git` and recommend cloning and `-e .` for developers?

Obviously getting it on PyPI would also make this easier